spec/dummy_app/app/models/person.rb

```diff
@@ -40,13 +40,9 @@ def load(value)
       end
     end
 
-    def dump(zone)
-      self.class.dump(zone)
-    end
+    delegate :dump, to: :class
 
-    def load(value)
-      self.class.load(value)
-    end
+    delegate :load, to: :class
```

Comment:
These `delegate` changes are not needed and should be reverted.

Reply:
same thing here